docs/references/resolvers.md

```diff
@@ -45,3 +45,45 @@ class Color:
             return list(self.fruits.objects.filter(...))
         return query()
 ```
+
+## Issues with Resolvers
+
+It is important to note that if you override resolvers, you will lose access to all of the things that come by default
+with the library (e.g. `Pagination`, `Filter`). On your root `Query`, you can just use a custom `get_queryset` to achieve
+similar results, while keeping pagination and filtering intact.
```

Comment:
What do you think?

```suggestion
similar results, while keeping pagination and filtering intact, but note that it will affect all root queries for that type.
```